library/Kubernetes/Web/PodDetail.php

```diff
@@ -131,29 +132,42 @@ protected function assemble(): void
                 null,
                 new HtmlElement('h2', null, new Text('Containers')),
                 new ContainerList($this->pod->container)
-            ),
-            new HtmlElement(
+            )
+        );
+
+        if (Auth::getInstance()->hasPermission(Auth::SHOW_PERSISTENT_VOLUME_CLAIMS)) {
+            Auth::getInstance()->applyRestrictions($this->pod->pvc, Auth::SHOW_PERSISTENT_VOLUME_CLAIMS);
+
+            $this->addHtml(new HtmlElement(
                 'section',
                 null,
                 new HtmlElement('h2', null, new Text($this->translate('Persistent Volume Claims'))),
                 new PersistentVolumeClaimList($this->pod->pvc)
-            ),
-            new HtmlElement(
+            ));
+        }
+
+        if (Auth::getInstance()->hasPermission(Auth::SHOW_EVENTS)) {
+            $events = Event::on(Database::connection())
```

Comment:
I think this can be harmonized with the other event queries, i.e.
```php
            $events = Event::on(Database::connection())
                ->filter(Filter::equal('referent_uuid', $this->pod->uuid));
```